perfkitbenchmarker/providers/openstack/os_disk.py

```diff
@@ -24,96 +24,198 @@
 
 FLAGS = flags.FLAGS
 
+ROOT = 'root'
```

Comment:
I think we call this disk type "boot" elsewhere in the code, so please consider that name. However, thanks to the disk refactor, OpenStack has its own disk type namespace, so if you think "root" is better, that's fine.